src/themes/mainTheme.ts

```diff
@@ -1,70 +1,72 @@
-import { createTheme, PaletteColorOptions, SxProps } from '@mui/material';
+import { createTheme, darken, PaletteColor, PaletteColorOptions, SxProps } from '@mui/material';
 import { enUS as coreEnUs, nbNO as coreNbNo, nnNO as coreNnNo } from '@mui/material/locale';
 import type {} from '@mui/x-date-pickers/themeAugmentation';
 import i18n from '../translations/i18n';
+import '@mui/material/Chip';
```

Comment:
Hvorfor?
Antar det kan være pga `declare module '@mui/material/Chip'`, men vi gjør ikke samme for `declare module '@mui/material/styles'` og `declare module '@mui/material/Button'`

```suggestion
```